test/modules/TestResultCoordinator/report/TestReportGeneratorFactory.cs

```diff
@@ -2,6 +2,8 @@
 namespace TestResultCoordinator.Report
 {
     using System;
+    using TestResultCoordinator.Storage;
+
     class TestReportGeneratorFactory : ITestReportGeneratorFactory
     {
         public TestReportGeneratorFactory()
```

Comment:
Make this constructor to take ITestOperationResultStorage instead of passing through Create method.